apps/src/templates/RegionalPartnerSearch.jsx

```diff
@@ -494,6 +481,56 @@ class RegionalPartnerSearch extends Component {
   }
 }
 
+const StartApplicationButton = ({
+  className,
+  id,
+  link,
+  partnerSite,
+  nominated,
+  priorityDeadlineDate
+}) => {
+  if (!link) {
+    link = studio('/pd/application/teacher');
+    if (nominated) {
+      link += '?nominated=true';
+    }
+  }
+  const target = partnerSite ? '_blank' : null;
+  const buttonText = partnerSite
+    ? "Apply on partner's site"
+    : 'Start application';
+
+  const notificationText = priorityDeadlineDate
+    ? `Priority deadline for your region is ${priorityDeadlineDate}. Sign up now to reserve your space!`
+    : 'We still have spaces at your local workshop! It’s not too late to sign up.';
+
+  return (
+    <div>
+      <Notification
+        type="information"
+        notice=""
+        details={notificationText}
+        dismissible={false}
+      />
+
+      <a className={className} id={id} target={target} href={link}>
+        <button type="button" style={styles.bigButton}>
+          {buttonText}
+        </button>
+      </a>
+    </div>
+  );
+};
+
+StartApplicationButton.propTypes = {
+  className: PropTypes.string,
```

Comment:
it looks like you are calling this prop `class` when you pass it in

Reply:
Good catch.